src/mem/ruby/system/VIPERSequencer.cc

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2013-2015 Advanced Micro Devices, Inc.
```

Comment:
Copyright should reflect UW adding this file, not AMD, since you are the author ... I guess you can include me if you want to, but definitely you and UW